packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -144,6 +146,43 @@ def test_if_we_have_capabilities(cluster):
     assert {'name': 'PACKAGE_MANAGEMENT'} in r.json()['capabilities']
 
 
+def test_cosmos_package_add(cluster):
+    r = cluster.post(
+        '/package/add',
+        headers={
+            'Accept': (
+                'application/vnd.dcos.package.add-response+json;'
+                'charset=utf-8;version=v1'
+            ),
+            'Content-Type': (
+                'application/vnd.dcos.package.add-request+json;'
+                'charset=utf-8;version=v1'
+            )
+        },
+        json={
+            'packageName': 'cassandra',
+            'packageVersion': '1.0.20-3.0.10'
+        }
+    )
+
+    user_config = load_json("/opt/mesosphere/etc/expanded.config.json")
+    if (user_config['cosmos_staged_package_storage_uri_flag'] and
+            user_config['cosmos_package_storage_uri_flag']):
+        # if the config is enabled then Cosmos should accept the request and
+        # return 202
+        assert r.status_code == 202, 'status = {}, content = {}'.format(
```

Comment:
Minor comment.

These assert lines could be folded into a single line as the fall under our 120 col limit.